inc/about.php

```diff
@@ -157,9 +157,13 @@ function cptui_about_page_newsletter() {
 	}
 
 	?>
-	<h3><?php esc_html_e( 'Stay informed', 'custom-post-type-ui' ); ?></h3>
```

Comment:
Here is a screenshot comparing the old version (on the top) with the new version (on the bottom).

- The "Stay informed" text has been removed
- The cptui-badge now sits lower than it did previously

![Screen Shot 2020-02-04 at 16 14 34 ](https://user-images.githubusercontent.com/5194588/73787564-91b64080-4769-11ea-9fd8-0ad20e04a6a4.png)